regression/validate-trace-xml-schema/check.py

```diff
@@ -61,7 +61,10 @@
     # the SAT back-end only
     ['integer-assignments1', 'test.desc'],
     # this test is expected to abort, thus producing invalid XML
-    ['String_Abstraction17', 'test.desc']
+    ['String_Abstraction17', 'test.desc'],
+    # these tests use characters that cannot be displayed in XML
+    ['ACSL', 'operators.desc'],
```

Comment:
:no_entry_sign: If the changes cause output to include more characters that are invalid XML then they should not be merged. The correct thing to do would be to escape these characters.

Reply:
For details of how escaping works in XML see - https://www.w3.org/TR/REC-xml/#syntax
I think the code for this escaping exists already. See `xmlt::escape_attribute` in `src/util/xml.cpp`.

Reply:
:point_up: What he said.  Let's not generate invalid XML.